scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -471,6 +471,7 @@ private void receivedServerHelloDone(ServerHelloDone message) throws HandshakeEx
 			generateKeys(premasterSecret);
 			break;
 		case PSK:
+			recordLayer.pauseRetransmission();
```

Comment:
> You seems to say, that pause/resume is not necessary for client side ? why just for client side ?

Oh no, I just got aware, that very short after the "resumeRetransmission" it's stopped again. But this is only true for the client side.

The other part, do also a "pause" for certificate bases handshake, was mainly influenced by issue #235 

So, if it's known, that there is potentially more time required to calculate then the timeout, we either pause or rely on working retransmission.